/**:
  ros__parameters:
    # Pressure sensors
    pressure_sensors_installed: False

    # IMU
    imu_timeout_duration: 0.5 # time without messages from the IMU till error is produced [s]

    # Motors
    motor_off_time: 30000000.0 # time of no use or updates when the hcm goes to soft off
    motor_timeout_duration: 0.5 # time without messages from the servos till error is produced [s]

    # Animations
    anim_server_wait_time: 10.0 # time the hcm waits for animation server
    animations:
      falling_back: "falling_back"
      falling_front: "falling_front"
      falling_left: "falling_left"
      falling_right: "falling_right"
      init: "init"
      stand_up_back: "stand_up_back"
      stand_up_front: "stand_up_front"
      startup: "startup"
      turning_front_left: "turning_front_left"
      turning_front_right: "turning_front_right"

    # Falling
    stand_up_active: true # Enables the robot to stand up automatically
    falling_active: true # Enables the check for falling and corresponding counter measurements
    # Threshold tighter --> earlier reaction but more false positives, e.g. while walking
    # Threshold softer --> too late reaction
    falling_thresh_gyro_pitch: 7.0 # > gyroY
    falling_thresh_gyro_roll: 7.0 # > gyroX
    falling_thresh_orient_roll: 60.0 # > Point of no return in degrees
    falling_thresh_orient_pitch: 45.0 # > Point of no return in degrees
    # Duration in seconds in which the robot has to be in a falling state to trigger the fall
    smooth_threshold: 0.04

    # Fallen
    fallen_orientation_thresh: 60.0 # Lean (degrees) after which we consider the robot fallen on this side
    fallen_angular_velocity_thresh: 0.2 # Angular velocity (rad/s) under which we consider the robot as not moving (the fall is over and we are lying on the ground)

    # Pick up
    pick_up_accel_threshold: 7.0
